proposed/http-handlers/request-handlers-meta.md

```diff
@@ -318,6 +318,51 @@ to define a standard that is specific to the nature of asynchronous middleware.
 _See "client vs server side middleware" in [relevant links](#8-relevant-links) for
 additional information._
 
+#### What is the role of the request handler?
+
+Middleware has the following roles:
+
+- Producing a response on its own. If specific request conditions are met, the
+  middleware an produce and return a response.
```

Comment:
I guess you mean "middleware can produce" :)

Reply:
Fixed!